Detections/SymantecVIP/ExcessiveFailedAuthenticationsfromInvalidInputs.yaml

```diff
@@ -6,7 +6,7 @@ severity: Medium
 requiredDataConnectors:
   - connectorId: SymantecVIP
     dataTypes: 
-      - Syslog
+      - SymantecVIP
```

Comment:
@chicduong - yeah, this should be Syslog.

Reply:
@shainw I'm confused on this, because based on our discussion in PR #750 (https://github.com/Azure/Azure-Sentinel/pull/750) the Dataypes should be the name of the table, since these alerts depend on a Kusto Function that would make the table name SymantecVIP instead of Syslog.

Reply:
That's because the metadata info operates on Syslog or CEF table or Custom Log table - this is the metadata info.
The Kusto Function itself is in the analytic query. This metadata info is used by the publish process which does not really take the parser into account (needs to know the underlying table being operated on). Hope this clarifies. 
@chicduong - this dataType should be Syslog - drop this change from the PR

Reply:
corrections have been made and pushed